v3/lints/etsi/lint_qcstatem_qctype_correct.go

```diff
@@ -21,30 +21,30 @@ import (
 	"github.com/zmap/zlint/v3/util"
 )
 
-type qcStatemQctypeWeb struct{}
+type qcStatemQctypeWebEsignEseal struct{}
 
 func init() {
 	lint.RegisterCertificateLint(&lint.CertificateLint{
 		LintMetadata: lint.LintMetadata{
-			Name:          "w_qcstatem_qctype_web",
-			Description:   "Checks that a QC Statement of the type Id-etsi-qcs-QcType features at least the type IdEtsiQcsQctWeb",
+			Name:          "e_qcstatem_qctype_web_esign_eseal",
```

Comment:
I believe that I would consider the name of the lint to actually be in the public interface of ZLint. Library usages pull up individual lints by name. Additionally, downstream automation may be parsing ZLint's JSON output and looking for a particular name

Now, the only reason I can think someone would do that is if the CA had a history of failing that particular lint, and thus have a particular interest in referring to it specifically. Regardless, however much more accurate the new name may be, I don't think the risk of breaking downstream automation is quite worth it.

```suggestion
			Name:          "e_qcstatem_qctype_web,
```